modules/orbitsoftBidAdapter.js

```diff
@@ -0,0 +1,232 @@
+
+        placementCode = bidObj.placementCode;
+
+        // Set the status
+        bidObj.status = CONSTANTS.STATUS.GOOD;
+      }
+
+      // @if NODE_ENV='debug'
+      utils.logMessage('JSONP callback function called for ad ID: ' + id);
+      // @endif
+
+      let bid = [];
+      if (jptResponseObj.cpm && jptResponseObj.cpm !== 0) {
+        // Store bid response
+        responseCPM = jptResponseObj.cpm;
+        // Bid status is good (indicating 1)
+        bid = bidfactory.createBid(CONSTANTS.STATUS.GOOD, bidObj);
+        bid.bidderCode = bidCode;
+        bid.cpm = responseCPM;
+        bid.adUrl = jptResponseObj.content_url;
+        bid.width = jptResponseObj.width;
+        bid.height = jptResponseObj.height;
+
+        // Styles params
+        let styles = utils.getBidIdParameter('style', bidObj.params);
+        let stylesParams = {};
+        for (let currentValue in styles) {
+          if (styles.hasOwnProperty(currentValue)) {
+            let currentStyle = styles[currentValue];
+            for (let field in currentStyle) {
+              if (currentStyle.hasOwnProperty(field)) {
+                let styleField = styleParamsToFieldsMap[currentValue + '.' + field];
+                if (styleField !== undefined) {
+                  stylesParams[styleField] = currentStyle[field];
+                }
+              }
+            }
+          }
+        }
+        bid.adUrl += '&' + utils.parseQueryStringParameters(stylesParams);
+
+        // Custom params
+        let customParams = utils.getBidIdParameter('customParams', bidObj.params);
+        let customParamsArray = {};
+        for (let customField in customParams) {
+          if (customParams.hasOwnProperty(customField)) {
+            customParamsArray['c.' + customField] = customParams[customField];
+          }
+        }
+        let customParamsLink = utils.parseQueryStringParameters(customParamsArray);
+        if (customParamsLink) {
+          // Don't append a "&" here, we have already done it in parseQueryStringParameters
+          bid.adUrl += customParamsLink;
+        }
+
+        // Remove the trailing "&"
+        bid.adUrl = removeTrailingAmp(bid.adUrl);
+
+        bidmanager.addBidResponse(placementCode, bid);
+      } else {
+        // No response data
+        // @if NODE_ENV='debug'
+        utils.logMessage('No prebid response from Orbitsoft for placement code ' + placementCode);
+        // @endif
+        // indicate that there is no bid for this placement
+        bid = bidfactory.createBid(CONSTANTS.STATUS.NO_BID, bidObj);
+        bid.bidderCode = bidCode;
+        bidmanager.addBidResponse(placementCode, bid);
+      }
+    } else {
+      // No response data
+      // @if NODE_ENV='debug'
+      utils.logMessage('No prebid response for placement');
+      // @endif
+    }
+  };
+
+  return {
+    callBids: baseAdapter.callBids,
+    setBidderCode: baseAdapter.setBidderCode,
+    buildJPTCall: buildJPTCall
+  };
```

Comment:
In order for your adapter to be alias-able the `.constructor` reference on your adapter instances needs to point to the correct constructor function; by returning a new object that reference will be incorrect.

You can solve this with:
```Javascript
return Obect.assign(this, {
     callBids: baseAdapter.callBids,
     setBidderCode: baseAdapter.setBidderCode,
     buildJPTCall: buildJPTCall
});
```

Reply:
@snapwich Fixed

Reply:
Sorry, you need to modify it to be the way I described (return `this`).  You can look at another adapter if you wish to get an example, but the modifications you made look like the old alias pattern.  `createNew` is no longer needed, but the constructor functions must return a reference to `this`.

Reply:
@snapwich Sorry for this. I made changes.